fairlearn/datasets/_fetch_boston.py

```diff
@@ -105,8 +112,23 @@ def fetch_boston(*, cache=True, data_home=None,
 
     References
     ----------
-    https://medium.com/@docintangible/racist-data-destruction-113e3eff54a8
-    https://github.com/scikit-learn/scikit-learn/issues/16155
+    .. [1] J. Vanschoren, “boston,” OpenML, 29-Sep-2014. [Online]. Available:
+       https://www.openml.org/d/531.
+
+    .. [2] D. Harrison and D. L. Rubinfeld, “Hedonic housing prices and the
+       demand for clean air,” Journal of Environmental Economics and Management,
+       vol. 5, no. 1, pp. 81–102, Mar. 1978.
+
+    .. [3] D. A. Belsley, E. Kuh, and R. E. Welsch, Regression diagnostics
+       identifying influential data and sources of collinearity. Hoboken, NJ,
+       NJ: Wiley-Interscience, 1980.
+
+    .. [4] J. Sykes, “- B 1000(Bk - 0.63)^2 where Bk is the proportion of blacks
+       by town · Issue #16155 · scikit-learn/scikit-learn,” GitHub, 18-Jan-2020.
+       [Online]. Available: https://github.com/scikit-learn/scikit-learn/issues/16155. # noqa: E501
```

Comment:
I think it's 2 whitespaces before `#` and one after (whenever using inline comments)

Reply:
Actually you don't need noqa here at all.